src/utils/commonFunctions.js

```diff
@@ -114,6 +114,10 @@ export const getStatistic = (data, type, statistic, perMillion = false) => {
     const recovered = data?.[type]?.recovered || 0;
     const migrated = data?.[type]?.migrated || 0;
     count = confirmed - deceased - recovered - migrated;
+  } else if (statistic === 'percentPositive') {
+    const confirmed = data?.[type]?.confirmed || 0;
+    const tested = data?.[type]?.tested || 0;
+    count = tested === 0 ? 0 : confirmed/tested*100;
```

Comment:
Hey, the CSS we can sort out, but the primary problem is this calculation. Testing data and confirmed counts may include backlogs and hence, this won't give the actual positivity rate. The solution is to modify the API to include the number of positive samples out of the total samples tested. I think most states report this data. I'll see how it can be added to the API.

Reply:
What do you mean by backlogs? Shouldn't that formula work fine since the confirmed cases and testing numbers are updated daily?

Reply:
Backlogs are when states reveal cases which were found to be positive at an earlier date but weren't reported in the state bulletin due to whatever reasons (mainly [bureaucratic](https://www.whatshot.in/bangalore/recorded-spike-bangalores-covid-19-accounting-error-c-23328)). 
> What do you mean by backlogs? Shouldn't that formula work fine since the confirmed cases and testing numbers are updated daily?

Unfortunately not. There's more nuance to it.

Reply:
Oh okay, got it. I don't see how that should matter much since this is supposed to be an estimate anyway, but I can help out with the API integration if needed.

Reply:
@shuklaayush Sorry to bother you, but do you have any updates on this? I think adding this graph for now and then updating it later if needed would suffice too since the cumulative graph should be a good enough estimate of the trend of the positivity rate.